src/types/catalogs-products.d.ts

```diff
@@ -8,7 +9,7 @@ export interface ProductResponse extends Identifiable {
     base_product: boolean
     base_product_id: string
     commodity_type: string
-    components: string[]
+    components: string[] | ProductComponents
```

Comment:
Regarding `components` field in catalog products interface. When we take a look at Docs we have type `string[]` for `v2` version and the new `ProductComponents` type for `pcm` version. I think we should support both. Please, let me know if i'm wrong.
https://documentation.elasticpath.com/commerce-cloud/docs/api/pcm/catalogs/releases/products/get-all-products-in-a-release.html
https://documentation.elasticpath.com/commerce-cloud/docs/api/pcm/catalogs/releases/products/get-a-product-in-a-release.html